config/routes.rb

```diff
@@ -31,5 +31,7 @@
   patch '/reviews/:id', to: "reviews#update"
   delete '/reviews/:id/delete', to: 'reviews#destroy'
 
-  get '/application/:id', to: 'applications#show'
+  get 'pets/:id/applications/new', to: 'applications#new'
```

Comment:
pets/:id to get our pet varible. After reading future stories, we might have to change this path